### DISCUSSION TITLE
Need Extension which is compatible with Breeze Theme for converting jpg/png images into webp format.

### DISCUSSION THREAD

**Discussion by JamesDenish** [breezefront/community#202] [1. Help]
Hello @vovayatsyuk 

Need your help to find out the extension for converting .jpg/.png images into webp format and it should also  compatible with breeze theme.

Kindly suggest if you have any extension of swissuplabs or any other vendor which provides both above functionality

Thanks in advance

**Comment by dimdimich** [comment]
Hi,

you can consider using https://swissuplabs.com/page-speed-magento-2.html or simply follow https://breezefront.com/docs/next-gen-images
